articles/sendgrid-dotnet-how-to-send-email.md

```diff
@@ -149,7 +149,83 @@ The following examples show how to send a message using the Web API.
             }
         }
     }
+    
+## How to: Send email from ASP .NET Core API using MailHelper class
 
+The below example can be used to send a single email to multiple persons from ASP .NET Core api using the `MailHelper` class of `SendGrid.Helpers.Mail` namespace. For this example we are using ASP .NET Core 1.0. 
+
+In this example the Api key has been stored in the `appsettings.json` file  which can be overridden from Azure portal as shown in above examples.
+
+The contents of `appsettings.json` file should look like below  -
+
+    {
+       "Logging": {
+       "IncludeScopes": false,
+       "LogLevel": {
+       "Default": "Debug",
+       "System": "Information",
+       "Microsoft": "Information"
+         }
+       },
+     "SENDGRID_API_KEY": "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    }
+
+As first step we need to add the below code in Startup.cs file of the .NET Core api project. This is required so that we can access the `SENDGRID_API_KEY` from the `appsettings.json` file by using dependency injection in the api controller. The `IConfiguration` interface can be injected at the constructor of controller after adding it in the `ConfigureServices` method below. The content of `Startup.cs` file looks like below after adding the required code -
+
+        public IConfigurationRoot Configuration { get; }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            // Add mvc here
+            services.AddMvc();
+            services.AddSingleton<IConfiguration>(Configuration);
+        }
+
+At the controller after injecting the `IConfiguration` interface we can use the `CreateSingleEmailToMultipleRecipients` method of the `MailHelper` class to send a single email to multiple recipients . It accepts one addtional boolean parameter named `showAllRecipients` . This parameter can be used to control whether email receipients will be able to see each others mail id who are in the `To` section of email. The sample code for controller should be like below 
```

Comment:
At the controller after injecting the `IConfiguration` interface we > At the controller, after injecting the `IConfiguration` interface, we

multiple recipients . > multiple recipients.

It accepts one addtional boolean parameter named `showAllRecipients` . > The method accepts one additional boolean parameter named `showAllRecipients`.

This parameter can be used to control whether email receipients will be able to see each others mail id who are in the `To` section of email. > This parameter can be used to control whether email recipients will be able to see each others email address in the `To` section of email header.

Reply:
ok i will modify it . Thanks for the input :)